spec/views/curation_concerns/base/_form_progress.html.erb_spec.rb

```diff
@@ -20,10 +20,13 @@
   end
 
   context "for a new object" do
-    before { assign(:form, form) }
-
     let(:work) { GenericWork.new }
 
+    it "shows the header set in the locale config in the sidebar" do
+      controller_name = "batch_uploads"
+      expect(t("sufia.#{controller_name}.progress.header")).to eql "Batch Create New Works"
```

Comment:
Hmmm, this test looks like it's testing the locale itself rather than the view. I'd expect to see `rendered` asked if it contains `"Batch Create New Works"`.